tools/executors/node/impl.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
```

Comment:
Why do we need to check in the generated `.js` file and `js.map`?

Reply:
Good question. It's because this is loaded as an NX executor, which doesn't support TypeScript at runtime.

Our choices were to:

1. Build and publish this as a stand-alone NPM package which we install in the monorepo.
2. Build the JS on every developer machine and CI.
3. Commit the JS.

We decided on #3 as the simplest option for this "one-off" thing, with the possibility to improve this later if we keep changing it.

Reply:
Ahh, I see. Understandable.